src/helpers.js

```diff
@@ -2,94 +2,96 @@
 
 const fs = require('fs');
 
-function readFile(file) {
-	return fs.readFileSync(file, 'utf8');
-}
+class Helpers {
+	static readFile(file) {
+		return fs.readFileSync(file, 'utf8');
+	}
 
-function hasProperties(obj, properties) {
-	let hasProperty = true;
+	static hasProperties(obj, properties) {
+		let hasProperty = true;

```

Comment:
nit: I think this variable name would be clearer as `hasAllProperties` since it looks like you set this to `false` if any property is missing from the object